common/names.go

```diff
@@ -56,14 +54,6 @@ const (
 	DiscoveryMetricsServingCert = "discovery-metric-serving-cert"
 )
 
-// GetLocalProvisionerImage return the image to be used for provisioner daemonset
-func GetLocalProvisionerImage() string {
-	if provisionerImageFromEnv := os.Getenv(ProvisionerImageEnv); provisionerImageFromEnv != "" {
```

Comment:
`ProvisionerImageEnv` reference should be removed in the `names.go` file.

Reply:
Done